scan/lib/scan/test_command_generator.rb

```diff
@@ -95,6 +95,14 @@ def xcodebuild_log_path
         return File.join(containing, file_name)
       end
 
+      # Generate destination parameters
+      def destination
+        unless Scan.cache[:destination]
+          Scan.cache[:destination] = [*Scan.config[:destination]].map { |dst| "-destination '#{dst}'" }.join(' ')
```

Comment:
Right. Elsewhere in this PR we are using `Array()` for this purpose, and I think it is a little easier to read/understand.